### ISSUE TITLE
Autostart web UX in Apache (or nginx)

### ISSUE THREAD

**Issue by federico3** [federico3/trovastelle#50]
We have a service for the arrow controller. The frontend and backend should also autostart. To do this, we need to figure out wsgi.

**Comment by federico3** [comment]
```
sudo su
cd /home/pi/trovastelle
python -m pip install -e .
python -m pip install -r requirements.txt
```

**Comment by federico3** [comment]
Resolved in 16dcb3de749207b42446db0f1a5d8f93f8484c04 and documented in 0673e7862d4ceeda29d9d380409a7795fbe0b62a
